interfaces/mount/spec_test.go

```diff
@@ -91,3 +92,27 @@ func (s *specSuite) TestSpecificationIface(c *C) {
 		{Name: "connected-plug"}, {Name: "connected-slot"},
 		{Name: "permanent-plug"}, {Name: "permanent-slot"}})
 }
+
+const snapWithLayout = `
+name: vanguard
+layout:
+  /usr:
+    bind: $SNAP/usr
+  /mytmp:
+    type: tmpfs
+    user: nobody
+    group: nogroup
```

Comment:
Maybe one more test with a numeric uid/gid?

Reply:
We don't allow those. Only "root" and "nobody" are allowed per London agreements for now.

Reply:
So to be clear, this would not validate.

Reply:
Why do we have code for it in https://github.com/snapcore/snapd/pull/4505/files#diff-4cc75dfc3d7110e4bc022dc5967af442R101 then? AIUI we support reading it and set an option? If we know we don't need it, maybe YAGNI ?

Reply:
Sorry, I was misreading the code. Ignore my previous comment https://github.com/snapcore/snapd/pull/4505#discussion_r162904204